Makefile

```diff
@@ -1,5 +1,10 @@
 MEASURE_DIRS := $(shell ls -d EXM_*)
 export SYNTHEA_DIR := synthea_output/$(shell date +%Y-%m-%dT%H%M%S)
+ifeq ($(strip $(CI_TOOL)),) 
+	BASE_DIR := "connectathon"
```

Comment:
```suggestion
	BASE_DIR := connectathon
```

These don't need to be in quotes, as they refer to directories that are recognized by bash. If they are in there we get weird output in the script like this:

![image](https://user-images.githubusercontent.com/16297930/78070991-af1f1980-736a-11ea-9d17-a133243cb3f8.png)